src/SignalR/common/Protocols.Json/src/Microsoft.AspNetCore.SignalR.Protocols.Json.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <Description>Implements the SignalR Hub Protocol using System.Text.Json.</Description>
     <TargetFrameworks>netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(DotNetBuildFromSource)' == 'true'">netcoreapp3.0</TargetFrameworks>
```

Comment:
Is this actually necessary since netcoreapp3.0 is already listed as a TargetFramework?

Reply:
This does the opposite, it excludes netstandard2.0.

Reply:
That makes more sense

Reply:
Is there a specific reason we're adding netcoreapp3.0 targets to DataProtection, Connection.Abstractions and others right now?

Reply:
I think I get it now. netcoreapp3.0 is the only valid target for source build.